bin/mc-send-to-console

```diff
@@ -12,8 +12,13 @@ if [ ! -p "${CONSOLE_IN_NAMED_PIPE}" ]; then
   exit 1
 fi
 
+distro=$(cat /etc/os-release | grep -E "^ID=" | cut -d= -f2 | sed -e 's/"//g')
 if [ "$(id -u)" = 0 ]; then
-  gosu minecraft bash -c "echo $* > '${CONSOLE_IN_NAMED_PIPE:-/tmp/minecraft-console-in}'"
+  if [[ $distro == alpine ]]; then
```

Comment:
I don't know enough here, but is there more than just this one command that needs to get flipped? I can do some searching too. :)

Reply:
I haven't looked at other scripts, I just came accross this one when using on own server.

Reply:
```bash
cdfn@cdfn-ryzen dev/docker-minecraft-server (fix/mc-send-to-console-alpine %) » grep -r "gosu"
scripts/start:    exec gosu ${runAsUser}:${runAsGroup} "${SCRIPTS:-/}start-configuration" "$@"
build/ubuntu/install-packages.sh:  gosu \
bin/mc-send-to-console:    exec gosu minecraft bash -c "echo $* > '${CONSOLE_IN_NAMED_PIPE:-/tmp/minecraft-console-in}'"
```
It looks like there are only 3 usages, first one is covered with check already, second one is installation script for debian based distros and third is corrected via this PR.

Reply:
The only other use of `gosu` is in `start`, which is already wrapped in a conditional:

https://github.com/itzg/docker-minecraft-server/blob/901b889f2d95c1a188070ca43d310bee4202d166/scripts/start#L48-L53